README.md

```diff
@@ -55,6 +55,13 @@ specifies one function:
                   Value :: term()) -> ok.
 ```
 
+### Running as part of own supervision tree
+
+You can run `vmstats` as part of your own supervision tree instead of relying on
+additional application by adding result of `vmstats:child_spec(Sink, BaseKey)`
+to your supervisor. For explanation of arguments look into
+[Configuration](#Configuration) `sink` and `base_key` descriptions.
```

Comment:
I would actually recommend against this practice, since the child spec should be chosen according to the supervision tree's needs. I'd rather it go undocumented and "just works" for elixir users.

Reply:
Removed docs.